src/Generation/SnippetGenerator.php

```diff
@@ -244,13 +252,15 @@ private function rpcMethodExampleBidiStreaming(SnippetDetails $snippetDetails):
                 $streamVar->writeAll($snippetDetails->rpcArguments),
                 PHP_EOL,
                 AST::inlineVarDoc(
-                    $context->type($snippetDetails->methodDetails->responseType),
+                    $context->type($responseType),
                     $elementVar
                 ),
                 AST::foreach($streamVar->closeWriteAndReadAll(), $elementVar)(
                     $this->buildPrintFCall(
                         'Element data: %s',
-                        "{$elementVar->toCode()}->serializeToJsonString()"
+                        $responseType->isClass()
+                            ? "{$elementVar->toCode()}->serializeToJsonString()"
+                            : $elementVar->toCode()
```

Comment:
Curious, can you share the example that necessitates this code? I think all LRO response types must be a `Message`, so I'm wondering when we would ever _not_ use `serializeToJsonString`.

Reply:
I think you're right, I put this here for safety just in case. If you prefer, I'm happy to remove it 👍 .

Reply:
Hm...I'm kind of in favor of removing it so as to reduce complexity since the case it handles is not actually possible. A `google.protobuf.Any` can only contain a `message`.

Reply:
All set on LRO! This comment is on the bidi sample, so it won't auto resolve.